integration/app/cmd_serve_test.go

```diff
@@ -117,3 +118,48 @@ func TestServeStargateWithCustomConfigFile(t *testing.T) {
 
 	require.NoError(t, isBackendAliveErr, "app cannot get online in time")
 }
+
+// TestServeStargateWithName tests serving a new chain scaffolded using a local name instead of a GitHub URI.
+func TestServeStargateWithName(t *testing.T) {
+	var (
+		env     = envtest.New(t)
+		appName = "sgblog5"
+		root    = env.TmpDir()
+	)
+
+	env.Exec("scaffold an app",
+		step.NewSteps(step.New(
+			step.Exec(
+				envtest.IgniteApp,
+				"scaffold",
```

Comment:
Can we still keep using the Scaffold() func in these tests?

We can refactor it like this;

```go
fmt.Sprintf("github.com/test/%s", appName)
```

Move the above code in Scaffold to another func like (not sure about the func name):

```go
func GitModuleName(name string) string {
   return fmt.Sprintf("github.com/test/%s", name)
}
```

Or even we could manually add `github.com/test/` to each test, I don't have a strong opinion.

Reply:
Yes, I was planning to discuss how to change the integration tests to make the scaffolding calls consistent, because right now `Env.Scaffold(appName)` is internally formatting the app name into a GitHub URI leaving the new namings out. The app name should be sent with the right format.

I lean more towards explicitly using the "github.com/test/" prefix for the app names before calling the `Env.Scaffold`.

I will start refactoring the integration tests accordingly.